XRTK-Core/ProjectSettings/ProjectVersion.txt

```diff
@@ -1,2 +1,2 @@
-m_EditorVersion: 2019.4.23f1
-m_EditorVersionWithRevision: 2019.4.23f1 (3f4e01f1a5ec)
```

Comment:
@FejZa Let's revert the project version back to 2020.4 LTS for now

Reply:
I am fine with that